README.md

```diff
@@ -19,8 +19,23 @@ in situations in which the `:focus-ring` pseudo-selector should match.
 
 `npm install --save wicg-focus-ring`
 
+_We recommend only using versions of the polyfill that have been published to npm, rather than
+cloning the repo and using the source directly. This helps ensure the version you're using is stable
+and thoroughly tested. If you do want to build from source, make sure you clone the latest tag._
```

Comment:
Suggestion: newline before "If"

Reply:
I'm going to add that after this change cuz I'm scared to touch this wonky rebase again :D